src/roiextractors/extractors/minian/miniansegmentationextractor.py

```diff
@@ -13,6 +13,7 @@
 import numpy as np
 import pandas as pd
 import zarr
+from neuroconv.utils import calculate_regular_series_rate
```

Comment:
It should be imported within the extractor and not at the top level anyway. 

Consider doing the same for zarr just for the sake of import time efficiency (so all of those things are not front loaded when someone does import roiextractors)